src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -315,3 +315,26 @@
 
     :else
     constants/mainnet-chain-ids))
+
+(defn total-owned-collectible
```

Comment:
so there is a utils file in collectibles. Can we move this functionality there? -

imo we should avoid namespaces like
.../common/utils.cljs
this file is very likely to just end up being a random bunch of unrelated code.

Better is 
src/status_im/contexts/wallet/common/utils/collectibles.cljs
src/status_im/contexts/wallet/common/utils/networks.cljs
etc

or at least src/status_im/contexts/wallet/collectibles/utils.cljs


Also @vkjr has been working on multiple collectibles recently too, perhaps there is already some functionality like this in place?

Reply:
Copied to `src/status_im/contexts/wallet/collectible/utils.cljs`